src/components/tooltip/Tooltip.tsx

```diff
@@ -71,6 +72,7 @@ const Tooltip: React.FC<TooltipProps> = ({
   placement,
   type,
   visible,
+  wrapperClassName,
```

Comment:
can't we just name it className? :)

Reply:
There is another prop called "className" that will be added to the tooltip itself.